huxley/core/constants.py

```diff
@@ -35,6 +35,7 @@ class ProgramTypes(Constants):
     CLUB = 1
     CLASS = 2
 
+
```

Comment:
extra space can remove

Reply:
See https://peps.python.org/pep-0008/#blank-lines

Reply:
We may want to have a separate check for linting. Either at the PR stage or at the push stage.

Lots of the linting errors can be autofixed by off the shelf software.

Reply:
@srisainachuri

Reply:
alright double space matches the style guide so we can keep